src/main/java/com/google/cloud/solutions/datalineage/extractor/ColumnLineageExtractorFactory.java

```diff
@@ -91,22 +90,10 @@ public QueryColumns outputColumns() {
   }
 
   /**
-   * Returns a set of applicable Extractors for the provided SQL query.
+   * Returns a set of applicable Extractors
    */
   public ImmutableSet<ColumnLineageExtractor> buildExtractors() {
-    return outputColumns()
```

Comment:
The extractor types and ordering is still necessary - this has just been temporarily disabled. One question is how extractors work in order (or nested?). An insert with `SET col = CONCAT(colA, colB)` will be parsed correctly with colA and colB properly set as parents of col, but the evaluation occurs after the DML statement evaluation and the type is a '$insert'. I think this just needs some extension to allow the ordering of extractors to change, as well as allowing an extractor registered on the table itself instead of columnType (I think this is necessary for DML, but still wrapping my head around it). Any advice appreciated.

Reply:
They work in order